packages/connector-alipay-native/src/index.ts

```diff
@@ -61,10 +61,10 @@ export class AlipayNativeConnector implements SocialConnector {
     }
   };
 
-  public getAuthorizationUri: GetAuthorizationUri = async () => {
+  public getAuthorizationUri: GetAuthorizationUri = async (_redirectUri, state) => {
```

Comment:
Seems we only used the second param here. Curious why

Reply:
If we really don't care about the first param, we can use `_`. e.g.
```
async (_, state) => 
```

Reply:
For native SDKs, they only need `state`. `redirectUri` is sometimes required by some social connectors as part of validation. I was thinking of changing the `GetAuthorizationUri`input to be `(state: string, redirectUri?: string)` (currently, is `(redirectUri: string, state: string)`).
Which implementation is preferred?

Reply:
If `state` is indeed a required param, then my choice would be exchanging their positions.